### PR TITLE
[lldb] Set the byte size when setting the triple on SwiftASTContext

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp [hunk 1]
```diff
@@ -2451,6 +2451,18 @@ bool SwiftASTContext::SetTriple(const llvm::Triple triple, Module *module) {
...
   }
+
+  if (triple.isArch64Bit())
+    m_pointer_byte_size = 8;
+  else if (triple.isArch32Bit())
+    m_pointer_byte_size = 4;
+  else if (triple.isArch16Bit())
+    m_pointer_byte_size = 2;
+  else {
+    LOG_PRINTF(LIBLLDB_LOG_TYPES, "Could not set pointer byte size");
```
Comment (adrian-prantl) [inline lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp]: Can you also log the triple?

**Review by adrian-prantl — APPROVED** [review]

**Comment by augusto2112** [comment]
@swift-ci test

**Review by adrian-prantl — APPROVED** [review]

**Comment by augusto2112** [comment]
`TestSwiftReferenceStorageTypes.py` is failing again on Linux

**Comment by augusto2112** [comment]
@swift-ci test

**Pull request merged**